packages/crud-store/README.md

```diff
@@ -0,0 +1,122 @@
+# RainCatcher Store Spike
+
+The store module for RainCatcher has the primary goal of providing users with a way to supply the data-accessing features required by other modules in whatever data storage engine they prefer.
+
+Since RainCatcher modules are expected to work with any supplied storage engine, they are forced to use a generic baseline API, and not leverage advanced features from a given database engine.
+
+Users, on the other hand, are not required to use it in their custom application code and can directly use their chosen data library and db engine features.
+
+We have the following options for a store module in WFM 1.0:
+
+## Per-module Repository pattern with slim CRUD Store
+
+In this alternative each of our modules would expose a Repository interface that would be part of their constructor-injected dependencies and have all the required data access functions.
```

Comment:
I think it's worth to start with this aproach when building wfm module as it will be still flexible to move to any other solution. This will also align with data service - raincatcher store will be used mostly to satisfy storage requirements for our modules - users can override this by providing their own Repository implementation. Customers who wants to save their own data may benefit from mongo connection available in cloud app. 

WDYT?